PlatformIO - error: #if with no expression  #if DEBUG_ON

I tried to use this library in my ESP32 project that is programmed with Visual Studio Code and PlatformIO

When I compile the project I get the following compiler error:

```
.pio/libdeps/esp32/ESP Telnet/src/DebugMacros.h:29:13: error: #if with no expression
 #if DEBUG_ON
             ^
```

my platformio.ini
```
[env:esp32]
platform = espressif32@5.4.0
board = upesy_wroom
framework = arduino
monitor_speed = 115200
monitor_filters = esp32_exception_decoder, colorize
build_type = release
board_build.partitions = min_spiffs.csv
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,6 +4,8 @@
 
 **Note:** Unreleased changes are checked in but not part of an official release (available through the Arduino IDE or PlatfomIO) yet. This allows you to test WiP features and give feedback to them.
 
+- changed order of macros in `DebugMacros.h` to avoid an error
+
 ## [2.2] – 2023-12-28
 
 - merged pull request [#56](https://github.com/LennartHennigs/ESPTelnet/pull/56)  that fixes issues #54 and #52 submitted by [Laxilef](https://github.com/Laxilef)

--- a/src/DebugMacros.h
+++ b/src/DebugMacros.h
@@ -26,7 +26,12 @@
 
 /* ------------------------------------------------- */
 
-#if DEBUG_ON
+#ifndef DEBUG_ON
+  #define DEBUG_INFO
+  #define DEBUG_MSG(x)
+  #define DEBUG_VAR(...)
+  #define DEBUG_WHERE
+#else 
   #if DEBUG_USE_SERIAL
     #define DEBUG_SER_INFO      DEBUG_SERIAL.print(DEBUG_PREFIX); DEBUG_SERIAL.println("Compiled " __DATE__ ", " __TIME__)
     #define DEBUG_SER_MSG(x)    DEBUG_SERIAL.print(DEBUG_PREFIX); DEBUG_SERIAL.println(x)
@@ -55,11 +60,6 @@
   #define DEBUG_MSG(x)          DEBUG_SER_MSG(x); DEBUG_TEL_MSG(x)
   #define DEBUG_VAR(...)        DEBUG_SER_VAR(__VA_ARGS__); DEBUG_TEL_VAR(__VA_ARGS__)
   #define DEBUG_WHERE           DEBUG_SER_WHERE; DEBUG_TEL_WHERE
-#else
-  #define DEBUG_INFO
-  #define DEBUG_MSG(x)
-  #define DEBUG_VAR(...)
-  #define DEBUG_WHERE
 #endif
 
 /* ------------------------------------------------- */

